static/index.html

```diff
@@ -125,11 +125,17 @@ <h4>
                     <ul>
                         <li><span class="bold">Non-provability checks:</span> a red turnstile <span class="turnstile not-provable">⊢</span> means that the sequent is known <span class="bold">not to be provable</span>, however a black turnstile is not claiming anything about provability.
                         <li><span class="bold">"Auto-reverse":</span> when switched on this option applies systematically reversible rules to generated sequents.
-                        This includes not only the introduction rules for <span class="flip">&</span>, &, ⊥, ⊤, ! and 1, but also:
+                            This includes not only the introduction rules for <span class="flip">&</span>, &, ⊥, ⊤, ! and 1, but also:
+                            <ul>
+                                <li>axiom rule</li>
+                                <li>dereliction rule on ?? formulas</li>
+                                <li>⊗ rule with empty context</li>
+                            </ul>
+                        <li><span class="bold">Automated prover:</span> you can <span class="bold">double-click on a turnstile ⊢</span> to launch the auto-prover on a sequent. Three possibilities:
                         <ul>
-                            <li>axiom rule</li>
-                            <li>dereliction rule on ?? formulas</li>
-                            <li>⊗ rule with empty context</li>
+                            <li>the prover makes it, the proof is automatically filled up;</li>
+                            <li>the prover stops after an exhaustive research, the non-provable sequent is marked accordingly (red turnstile <span class="turnstile not-provable">⊢</span>);</li>
+                            <li>the prover is stopped after an arbitrary amount of time, we can not conclude on the provability of the sequent (orange turnstile <span class="turnstile not-auto-provable">⊢</span>).</li>
```

Comment:
```suggestion
                            <li>the prover is stopped by time-out, it can not conclude on the provability of the sequent (orange turnstile <span class="turnstile not-auto-provable">⊢</span>).</li>
```